examples/basic-tracer-node/index.js

```diff
@@ -46,7 +46,7 @@ function doWork(parent) {
   // Start another span. In this example, the main method already started a
   // span, so that'll be the parent span, and this will be a child span.
   const span = tracer.startSpan('doWork', {
-    parent,
+    parent : parent,
```

Comment:
This is not strictly required. In JS

```
const parent = getParent();
const obj = {
  parent
}
```

is the same as

```
const parent = getParent();
const obj = {
  parent: parent
}
```

Reply:
Thanks for clearing that up, did not know that.